src/universal/modules/meeting/components/LoadableNewMeetingAvatarMenu.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+import React from 'react';
+import {DEFAULT_MENU_HEIGHT, DEFAULT_MENU_WIDTH, HUMAN_ADDICTION_THRESH, MAX_WAIT_TIME} from 'universal/styles/ui';
+import Loadable from 'react-loadable';
+import LoadableLoading from 'universal/components/LoadableLoading';
+
+const LoadableNewMeetingAvatarMenu = Loadable({
+  loader: () => System.import(
+    /* webpackChunkName: 'NewMeetingAvatarMenu' */
+    'universal/modules/meeting/components/NewMeetingAvatarMenu'
+  ),
+  loading: (props) => <LoadableLoading {...props} height={DEFAULT_MENU_HEIGHT} width={DEFAULT_MENU_WIDTH} />,
+  delay: HUMAN_ADDICTION_THRESH,
```

Comment:
What is human addiction??

Reply:
Doherty threshold is 400ms, That was established in 1982. http://www.vm.ibm.com/devpages/jelliott/evrrt7.html
 I think we've become more impatient since then.